pom.xml

```diff
@@ -24,7 +24,7 @@
 
     <dependencies>
         <dependency>
-            <groupId>eu.darkbot</groupId>
+            <groupId>com.github.darkbot-reloaded</groupId>
```

Comment:
Could you delete this and leave only the change you made?

Reply:
i will revert it, i don't remember why did this happen or why i did this